include/eve/module/core/regular/frexp.hpp

```diff
@@ -41,35 +40,30 @@ struct frexp_t : elementwise_callable<frexp_t, Options, pedantic_option, raw_opt
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< eve::floating_value T >
-//!      eve::zipped<T,T> frexp(T x) noexcept;
+//!      template< eve::floating_value T >  eve::zipped<T,T> frexp(T x) noexcept;           //1
+//!      template< eve::floating_value T >  eve::zipped<T,T> frexp[pedantic](T x) noexcept; //2
 //!   }
 //!   @endcode
 //!
 //!   **Parameters**
 //!
-//!     * `x` :  [argument](@ref eve::value).
+//!     * `x` :  [floating value](@ref floating_value).
 //!
 //!   **Return value**
 //!
-//!     Computes the [elementwise](@ref glossary_elementwise) ieee frexp of the floating value,
-//!     returning a pair `{m,e}`  of values ` of the same type as `x`, which are related by
-//!     \f$x =  m\times 2^e\f$, with  \f$|m| \in [0.5, 1.5[\f$.
+//!     1. returns a pair `{m,e}`  of values ` of the same type as `x`, which are related by
```

Comment:
a kumi::tuple or else ppl will try to assign it to std::pair

Reply:
done